test/e2e/multi_namespace_test.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"log"
+	"testing"
+
+	monv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func TestMultinamespacePrometheusRule(t *testing.T) {
+	t.Parallel()
+
+	nsName := "openshift-test-prometheus-rules"
+
+	err := f.OperatorClient.CreateOrUpdateNamespace(&v1.Namespace{
```

Comment:
Given that the namespace name is not randomized, would you mind also removing it at the end of the tests, making sure two consecutive tests run on personal clusters don't influence each other.

Reply:
https://github.com/openshift/cluster-monitoring-operator/pull/263